data/roles.js

```diff
@@ -629,157 +648,163 @@ const roleData = {
+                "Chooses to sacrifice self once per game to convert another player to Mafioso.",
             ],
         },
         "Necromancer": {
             alignment: "Mafia",
             description: [
-                "Once per game, visits one dead person during the night.",
-                "That person will be resurrected.",
+                "Visits a dead player during the night once per game.",
+                "That player will be resurrected the following day.",
+                "If players identity was revealed upon death, they will remain revealed when ressurected."
             ],
             graveyardParticipation: "all",
         },
         "Mummy": {
             alignment: "Mafia",
             description: [
-                "Everyone who visits the mummy while the mummy is dead dies.",
+                "Everyone who visits the mummy while the mummy is dead, dies.",
             ],
         },
         "Poltergeist": {
             alignment: "Mafia",
             description: [
-                "Once dead, may visit one person a night and roleblock them.",
+                "Once dead, visits one player each night and roleblock them.",
            ],
            graveyardParticipation: "self",
         },
         "Informant": {
             alignment: "Mafia",
             description: [
-                "Can pick a person every night.",
-                "Any system messages that person may have got are given to the informant as well"
+                "Chooses a player each night and views any system messages they recieve the following day."
             ],
          },
         "Jinx": {
             alignment: "Mafia",
             description: [
-                "Each night, curses a player with a word.",
+                "Curses a player with a forbidden word each night.",
                 "If the person speaks the word the next day, they will die."
             ],
         },
         "Clown": {
             alignment: "Mafia",
             description: [
-                "Appears as Fool to self.",
+                "Fools around at night, visiting another player.",
                 "Will kill their visit target.",
+                "Appears as Fool to self."
             ],
         },
         "Graverobber": {
             alignment: "Mafia",
             description: [
-                "Can visit a dead player every night.",
-                "Learns the role of that player and takes all items they're holding.",
+                "Visits a dead player every night.",
+                "Learns the role of that player and takes any items they were holding.",
             ],
         },
         "Medusa": {
             alignment: "Mafia",
             description: [
-                "Once per game, during the day, can turn all visitors last night to stone.",
+                "Chooses to turn last nights visitors to stone, once per game, during the day.",
```

Comment:
Grammar. Should be "last night's". Still, maybe should be worded to "all visitors from previous night." ? Not necessary to change that way, just the way I see it.